sources/packages/web/src/views/institution/student/InstitutionStudentRestrictions.vue

```diff
@@ -1,3 +1,25 @@
 <template>
-  <tab-container><div>Restrictions component to be built.</div></tab-container>
+  <tab-container
+    ><student-restrictions :studentId="studentId"></student-restrictions
+  ></tab-container>
 </template>
+
+<script lang="ts">
+import { defineComponent } from "vue";
+import StudentRestrictions from "@/components/common/students/StudentRestrictions.vue";
+
+export default defineComponent({
+  components: { StudentRestrictions },
+  props: {
+    studentId: {
+      type: Number,
+      required: true,
+    },
+  },
+  setup() {
```

Comment:
Is there a need to have the `setup` at all?

Reply:
I am not sure of another way to return the `StudentRestrictions` component so that it can be used in the template.

Reply:
https://github.com/bcgov/SIMS/pull/1924/files#r1186383727

Reply:
Got it. Thanks Ann